docs/Reference/Routes.md

```diff
@@ -396,6 +396,29 @@ and route parameters are supported as well. Be aware that if you use
 [`fastify-plugin`](https://github.com/fastify/fastify-plugin) this option will
```

Comment:
@climba03003 This part of the docs I find very confusing, and I would like to clarify it as well. Does it mean that it will only not work if your routes are wrapped with fastify-plugin? Or if you use it _anywhere_ in your app, for _any_ plugins, it will break?

Reply:
There is a work around https://github.com/fastify/help/issues/174

Reply:
Incorporated https://github.com/fastify/help/issues/174, made wording around fastify-plugin less ambiguous, could you please recheck?